client-src/elements/chromedash-admin-blink-component-listing.ts

```diff
@@ -1,8 +1,19 @@
-import {html, css, LitElement} from 'lit';
+import {html, css, LitElement, TemplateResult} from 'lit';
 import {SHARED_STYLES} from '../css/shared-css.js';
 import {VARS} from '../css/_vars-css.js';
 import {LAYOUT_CSS} from '../css/_layout-css.js';
+import {customElement, property, query} from 'lit/decorators.js';
+import '../js-src/openapi-client';
+import {DefaultApiInterface} from 'chromestatus-openapi';
+import {get} from 'http';
 
+interface User {
```

Comment:
From the call site (https://github.com/GoogleChrome/chromium-dashboard/blob/e497b9f5b42603d4bc5e9a5a2a6fcbd62453cdb8/client-src/elements/chromedash-admin-blink-page.js#L79-L80), it looks like this is `import type {ComponentsUser} from 'chromestatus-openapi;`